docs/src/docs/hooks/use-focus-return.mdx

```diff
@@ -15,7 +15,7 @@ import { ModalDemos } from '@mantine/demos';
 
 ## Usage
 
-use-focus-return automatically manages focus returning to last focused element when given condition is met.
+`use-focus-return` automatically manages focus returning to last focused element when given condition is met.
```

Comment:
Hook name should not be wrapped with code, see other hooks

Reply:
Yeah, you're right. My bad.